.github/workflows/build.yaml

```diff
@@ -94,3 +94,53 @@ jobs:
           echo "::group::Log from these tests"
           [ ! -f Testing/Temporary/LastTest.log ] || cat Testing/Temporary/LastTest.log
           echo "::endgroup::"
+
+
+  windows-mingw:
+    runs-on: windows-latest
+    defaults:
+      run:
+        shell: msys2 {0}
+    
+    steps:
+      - uses: actions/checkout@v4
+      
+      - uses: msys2/setup-msys2@v2
+        with:
+          msystem: MINGW64
+          update: true
+          install: >-
+            git
+            mingw-w64-x86_64-gcc
+            mingw-w64-x86_64-gcc-fortran
+            mingw-w64-x86_64-cmake
+            mingw-w64-x86_64-openblas
+            mingw-w64-x86_64-parmetis
+            mingw-w64-x86_64-msmpi
+            base-devel
+      
+      - name: Configure
+        run: |
+          mkdir build
+          cd build
+          cmake .. -G "MSYS Makefiles" \
+            -DCMAKE_BUILD_TYPE="Release" \
+            -DBLA_VENDOR="OpenBLAS" \
+            -DWITH_OpenMP=ON \
+            -DWITH_LUA=ON \
+            -DWITH_Zoltan=OFF \
+            -DWITH_Mumps=OFF \
+            -DCREATE_PKGCONFIG_FILE=ON \
+            -DWITH_MPI=ON
```

Comment:
I accidentally commented on the commit instead of on the PR. So, repeating here. Sorry for the noise.

``` suggestion
           -DWITH_MPI=ON \
           -DMPIEXEC_EXECUTABLE="C:/Program Files/Microsoft MPI/Bin/mpiexec.exe" \
```

But for this to actually pass in CI, you'd need a new step to install MSMPI.
To install it locally, you can get to the installer from here: https://www.microsoft.com/en-us/download/details.aspx?id=105289 (The MS server seems to be down currently.)

I haven't looked into how that could be installed in CI though...

PS: In any case, you are missing a line continuation here.